packages/accordions/src/elements/accordion/components/Header.tsx

```diff
@@ -76,7 +75,7 @@ const HeaderComponent = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement
           onMouseOver: composeEventHandlers(onMouseOver, () => setIsHovered(true)),
           onMouseOut: composeEventHandlers(onMouseOut, () => setIsHovered(false)),
           ...other
-        })}
+        } as any) as any)}
```

Comment:
prefer `{...getHeaderProps() as HTMLAttributes<HTMLDivElement>}` vs overuse of `any`.